TOC.md

```diff
@@ -148,6 +148,7 @@
         - [`SET`](/data-type-string.md#set-type)
       - [JSON Type](/data-type-json.md)
     + Functions and Operators
+<<<<<<< HEAD
```

Comment:
Please resolve this conflict by referring to https://github.com/pingcap/docs-cn/pull/3865/files#diff-12810a4235487caeff6e7338176dc036R290. @zz-jason